WebApp/autoreduce_webapp/autoreduce_webapp/models.py

```diff
@@ -16,9 +16,9 @@ class Cache(models.Model):
     """
     created = models.DateTimeField(auto_now_add=True, blank=False)
 
-    def __unicode__(self):
-        # pylint: disable=no-member
-        return u'%s' % self.id_name
+    # pylint:disable=no-member
```

Comment:
Do we need this when it's __str__ and not __unicode__?

Reply:
Yes we do